src/localization/generalStrings.json

```diff
@@ -71,6 +72,7 @@
     "or": "ou",
     "expiry_date": "date d'expiration",
     "quantity": "Quantité",
+    "to": "à",
```

Comment:
Probably fine, but in general probably best not to rely on constructing translations from articles, prepositions etc. (the, to, from etc.) because they can change with context in different languages.

Reply:
true - I did check with alain with this one if that makes it better? :(